tests/test_cli.py

```diff
@@ -31,11 +31,11 @@
 import re
 
 VERSIONS_OUTPUT = """\
-  Installed: pepper-0:20-0.x86_64 at 1970-01-01 00:00
-  Built    :  at 1970-01-01 00:00
+  Installed: pepper-0:20-0.x86_64 at Thu Jan  1 00:00:00 1970
```

Comment:
I hope that tests run with LANG=C.
This is example from my PC:
LANG=C:  "**Wed May 17 14:56:53 2017**"
LANG=en_US: "**Wed 17 May 2017 02:56:53 PM CEST**"

So hardcoded datetime string in test_cli.py is in LANG=C format as you propose.

Another approach is to generate a datetime string during the test and not to use a hardcoded string. Do you want this change?